cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaBrokerConfigurationDiff.java

```diff
@@ -239,4 +239,27 @@ private void removeProperty(Map<String, ConfigModel> configModel, Collection<Alt
     public boolean isEmpty() {
         return brokerConfigDiff.isEmpty();
     }
+
+    /**
+     * @param entry tested ConfigEntry
+     * @param scope scope to test
+     * @return true if the entry matches the scope
+     */
+    private boolean isScope(ConfigEntry entry, Scope scope) {
+        return configModel.get(entry.name()).getScope().equals(scope);
+    }
+
+    /**
+     * Returns configuration difference only for parameters in the specified scope
+     * @return Collection of AlterConfigOp containing difference between current and desired configuration for the specified scope
+     */
+    protected Collection<AlterConfigOp> getConfigDiff(Scope scope) {
+        Collection<AlterConfigOp> clusterWideConfigDiff = new ArrayList<>();
```

Comment:
Since this method is also used for perBrokerConfigDiff, should `clusterWideConfigDiff` be named more generically?